packages/extension-chakra-storefront/src/components/header/index.jsx

```diff
@@ -275,23 +275,25 @@ const Header = ({
                                                         {navLinks.map((link) => {
                                                             const LinkIcon = link.icon
                                                             return (
-                                                                <List.Item
-                                                                    key={link.name}
-                                                                    value={link.name}
-                                                                >
-                                                                    <Link
-                                                                        useNavLink={true}
-                                                                        to={`/account${link.path}`}
+                                                                <List.Item key={link.name}>
+                                                                    <Button
+                                                                        asChild
+                                                                        variant="menu-link"
                                                                         css={styles.menuAccountLink}
```

Comment:
This will allow customised style from parent and not affect all buttons of this variant